pages/about.html

```diff
@@ -4,9 +4,53 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="../stylesheets/about.css">
 </head>
 <body>
-  
+  <div class="container">
+    <header class="header">
+      <div class="logo">
+        <a href="../pages/index.html"> <img src="../images/MM Logo.png" alt="logo"></a>
+      </div>
+      <nav class="nav">
+        <div class="nav_home"><a href="../pages/index.html">Home</a></div>
+        <div class="nav_about"><a href="../pages/about.html">About</a></div>
+        <div class="nav_portfolio"><a href="../pages/portfolio.html">Portfolio</a></div>
+        <div class="nav_contact"><a href="../pages/contact.html">Contact</a></div>
+      </nav>
+    </header>
+    <main class="main">
+      
+      <section class="about_me">
+        <h1>Passion</h1>
+        <h3>Software Engineering</h3>
```

Comment:
In general we don't want to skip from an h1 to and h3. We can use a class name to apply the styling we are looking for. 

```suggestion
        <h2 class="passion-detail">Software Engineering</h2>
```